### ISSUE TITLE
Error at quest pickup

### ISSUE THREAD

**Issue by Gemini1661** [Questie/Questie#5895]
## Bug description
Error generated every time a quest is accepted.

## Details
```
47x Questie/Modules/Tracker/AutoCompleteFrame.lua:69: attempt to index upvalue 'autoCompleteFrame' (a nil value)
[string "@Questie/Modules/Tracker/AutoCompleteFrame.lua"]:69: in function `CheckAutoCompleteQuests'
[string "@Questie/Modules/QuestieEventHandler.lua"]:95: in function <...rfaceQuestie/Modules/QuestieEventHandler.lua:92>
[string "=[C]"]: ?
[string "@Ace3/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:19: in function <...Ons/Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:15>
[string "@Ace3/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:54: in function `Fire'
[string "@Ace3/AceEvent-3.0-4/AceEvent-3.0.lua"]:120: in function <Ace3/AceEvent-3.0/AceEvent-3.0.lua:119>
[string "=[C]"]: in function `AcceptQuest'
[string "@Blizzard_UIPanels_Game/TBC/QuestFrame.lua"]:790: in function <...aceBlizzard_UIPanels_Game/TBC/QuestFrame.lua:783>

Locals:
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to index upvalue 'autoCompleteFrame' (a nil value)"
AutoCompleteFrame = <table> {
 private = <table> {
 }
 CheckAutoCompleteQuests = <function> defined @Questie/Modules/Tracker/AutoCompleteFrame.lua:63
 ShowAutoComplete = <function> defined @Questie/Modules/Tracker/AutoCompleteFrame.lua:56
 Initialize = <function> defined @Questie/Modules/Tracker/AutoCompleteFrame.lua:9
}
autoCompleteFrame = nil
```

## Questie & WoW version
Cataclysm Classic
Addon version 10.0.2

**Comment by MyTechnoHunter** [comment]
Fixed for next release (hotfix)

**Comment by BreakBB** [comment]
Yeah, fixed in 10d14897fb2d4de239717edca4ef9e182bdf97b0 and will be released today. Thanks for reporting this @Gemini1661
